src/test-utils/constants.tsx

```diff
@@ -0,0 +1,29 @@
+import { CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { FeeAmount, Pool } from '@uniswap/v3-sdk'
+import JSBI from 'jsbi'
+
+export const testToken1 = new Token(1, '0x0000000000000000000000000000000000000001', 18)
+export const testToken2 = new Token(1, '0x0000000000000000000000000000000000000002', 18)
+export const testToken3 = new Token(1, '0x0000000000000000000000000000000000000003', 18)
+export const testRecipientAddress = '0x0fF2D1eFd7A57B7562b2bf27F3f37899dB27F4a5'
```

Comment:
Why is this a real address? Should it not be something like:

```suggestion
export const testRecipientAddress = '0x0000000000000000000000000000000000000004'
```